internal/model/instances/instances.proto

```diff
@@ -0,0 +1,34 @@
+/**
```

Comment:
Would it make sense to flatten the directory structure in `internal/model`?

Reply:
Or were these split up because there name collisions in the generated code?

Reply:
yeah I put them in separate directories because I wanted to avoid name collisions in the future. 
For example, messages named `Meta` or `Type` might be used in other messages in the future but have different contents.